### ISSUE TITLE
Is everything going OK with the course?

### ISSUE THREAD

**Issue by willpearse** [pearselab/r-intro-TARobison#1]
I notice you haven't done a lot in this repo... Would you like to schedule a meeting?

**Comment by TARobison** [comment]
Yes, I'm auditing the course so I'm just doing the reading and as many of the exercises as I can, but I haven't uploaded anything. Thanks for checking in!
